Source/WebKit/Scripts/webkit/tests/SerializedTypeInfo.cpp

```diff
@@ -115,7 +119,7 @@ namespace WebKit {
 
 template<typename E> uint64_t enumValueForIPCTestAPI(E e)
 {
-    return static_cast<std::make_unsigned_t<std::underlying_type_t<E>>>(e);
+    return unsignedCast(enumToUnderlyingType(e));
```

Comment:
I think if we removed the `requires` from the implementation of `unsignedCast` we could do this with `unsignedCast` alone because `make_unsigned` handles this case.
```suggestion
    return unsignedCast(e);
```